### PR TITLE
Automate several Godling archetype features

### PR DESCRIPTION
**Pull request by pedrogrullada**

### CODE REVIEW

File: packs/feat-effects/effect-absolve-sins.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        "rules": [
+            {
+                "keep": "lower",
+                "key": "RollTwice",
+                "predicate": [
+                    "item:slug:absolve-sins"
+                ],
+                "selector": "will"
+            }
+        ],
```
Comment (TikaelSol) [inline packs/feat-effects/effect-absolve-sins.json:32]: The description says you adjust the DoS, but the rules are misfortune

File: packs/feat-effects/effect-divine-invulnerability.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            "remaster": true,
+            "title": "Pathfinder War of Immortals"
+        },
+        "rules": [
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Godling.DivineInvulnerability.IncreasedResistanceToggle",
+                "option": "divine-invulnerability-increased-resistance",
+                "toggleable": true
+            },
```
Comment (TikaelSol) [inline packs/feat-effects/effect-divine-invulnerability.json:29]: I hate having two effects, but this second part is a timed thing so we should do an effect for it. Maybe we can ItemAlteration the link into the description of domain spells? Though, maybe impossible to do that properly currently so a potential future thing.

Comment (pedrogrullada) [reply packs/feat-effects/effect-divine-invulnerability.json]: Effect added. The note would be a good idea, I would need to think about it some more, but at first glance it doesn't seem like we can narrow down the predicate for it quite enough

File: packs/feats/receive-prayers.json [hunk 3]
```diff
@@ -28,11 +28,23 @@
...
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "receive-prayers",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "action:sense-motive"
```
Comment (simonward) [inline packs/feats/receive-prayers.json:40]: Did you intend to include `"receive-prayers"` in the predicate?

File: packs/feats/absolve-sins.json [hunk 4]
```diff
@@ -7,11 +7,11 @@
...
+            "value": "<p>Your forgiveness is without limit, though the path to penitence can be painful, especially against those who have trespassed against your current mortal form. Spend a Mythic Point; each enemy within a @Template[type:emanation|distance:30] takes @Damage[14d6[mental]|options:area-damage] damage (@Check[will|against:class-spell|basic|options:area-effect] save against your class DC or spell DC, whichever is higher) as you force them to reflect upon their sins and misdeeds.</p>\n<p>Any creature who dealt damage to you or your hierophant since the end of your previous turn uses the outcome one degree of success worse than the result of their saving throw.</p>"
```
Comment (simonward) [inline packs/feats/absolve-sins.json:14]: Did you intend to link to the Absolve Sins effect?

Comment (pedrogrullada) [reply packs/feats/absolve-sins.json:14]: I linked it and didn't push the change (:

File: packs/feats/receive-prayers.json [hunk 5]
```diff
@@ -28,11 +28,23 @@
...
+                ],
+                "selector": "perception",
```
Comment (pedrogrullada) [inline packs/feats/receive-prayers.json:42]:
```suggestion
                "predicate": [
                    "action:sense-motive",
                    "receive-prayers"
                ],
                "selector": "perception",
```

File: packs/feats/receive-prayers.json [hunk 6]
```diff
@@ -28,11 +28,23 @@
             "remaster": true,
             "title": "Pathfinder War of Immortals"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "receive-prayers",
+                "toggleable": true
+            },
```
Comment (pedrogrullada) [inline packs/feats/receive-prayers.json:36]:
```suggestion
            {
                "domain": "perception",
                "key": "RollOption",
                "option": "receive-prayers",
                "toggleable": true
            },
```

File: packs/feats/receive-prayers.json (hunk 3 above)
Comment (pedrogrullada) [reply packs/feats/receive-prayers.json]: I did, my bad.

**Pull request merged**